js-miniapp-sample/src/pages/universal-bridge.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import MiniApp from 'js-miniapp-sdk';
+import { HostAppEvents } from 'js-miniapp-sdk';
+
+import {
+  Button,
+  TextField,
+  CardContent,
+  CardActions,
+  makeStyles,
+} from '@material-ui/core';
+
+import GreyCard from '../components/GreyCard';
+
+const useStyles = makeStyles((theme) => ({
+  scrollable: {
+    overflowY: 'auto',
+    width: '100%',
+    paddingTop: 20,
+    paddingBottom: 20,
+  },
+  card: {
+    width: '100%',
+    height: 'auto',
+  },
+  actions: {
+    justifyContent: 'center',
+  },
+  content: {
+    justifyContent: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    fontSize: 18,
+    color: theme.color.primary,
+    fontWeight: 'bold',
+    paddingBottom: 0,
+  },
+  textfield: {
+    width: '90%',
+    maxWidth: 300,
+    marginTop: 10,
+    marginBottom: 10,
+    background: 'white',
+    '& input': {
+      color: theme.color.primary,
+      lineHeight: '1.5em',
+      fontSize: '1.2em',
+    },
+  },
+}));
+
+const UniversalBridge = () => {
+  const classes = useStyles();
+  const defaultJsonValue = '{"data":"This is a sample json information"}';
+  let inputValue = defaultJsonValue;
+  let [receiveJsonInfo, setReceiveJsonInfo] = useState('');
+
+  window.addEventListener(HostAppEvents.RECEIVE_JSON_INFO, function (e) {
+    let message = e.detail.message;
+    console.log(message);
+    receiveJsonInfo = message;
+    setReceiveJsonInfo(message);
+    receiveJsonResponseBack();
+  });
+
+  const handleInput = (e: SyntheticInputEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    inputValue = e.currentTarget.value;
+  };
+
+  const sendJson = () => {
+    const info = { content: inputValue };
+    MiniApp.universalBridge
+      .sendJsonToHostapp(info)
+      .then((success) => {
+        console.log(success);
+      })
+      .catch((miniAppError) => {
+        console.error('Send Json Error: ', miniAppError);
+      });
+  };
+
+  const receiveJsonResponseBack = () => {
+    const responseReceiveJson = { content: 'Json received successfully' };
+    MiniApp.universalBridge
+      .sendJsonToHostapp(responseReceiveJson)
+      .then((success) => {
+        console.log(success);
+      })
+      .catch((miniAppError) => {
+        console.error('Receive Json Response Error: ', miniAppError);
+      });
+  };
```

Comment:
I don't know this is the correct way to send back the response back to host app, because i have suggested @rleojoseph about this to get the callback through `messageBridge` and he was kind a suggested not to do like this, and also `sendJsonToHostapp` should be a different one, otherwise both event will call the same one to bridge, it will hard to differentiate later on.

Reply:
@khairul-alam-licon , we should send back the response/callback in the same interface not using a new one. But as of now, we won't be sending any response back to the Host app for Interface 2. So update the code accordingly. I will update the doc as well

Reply:
Thank you for the discussion, I've updated accordingly.